feature-libs/cart/base/components/cart-shared/cart-item-list-row/cart-item-list-row.component.html

```diff
@@ -48,6 +48,14 @@
 
       <!-- Promotion -->
       <cx-promotions [promotions]="item.promotions"></cx-promotions>
+
+      <!-- Item Delivery Details Outlet -->
+      <ng-container *cxFeatureLevel="'5.1'">
```

Comment:
![Screenshot 2023-01-11 at 4 09 29 PM](https://user-images.githubusercontent.com/44440575/211907679-6b27675a-80bb-4123-aa49-b433099d44d0.png)
be careful when there is another cart details addon stuff